### ISSUE TITLE
Converting Huggingface Seq2SeqLMOutput object to (aspect, sentiment) form

### ISSUE THREAD

**Issue by RajNS02** [kevinscaria/InstructABSA#3]
Hello, 

Using Huggingface pre-trained transformer, I am getting the output of:
'The cab ride was amazing but the service was pricey'  in the form of Seq2SeqLMOutput. I want to convert this in the form of (aspect, sentiment).
I tried using the `model.decode(tokenizer.decode(predicted_output.logits[0], skip_special_tokens=True)`, but I am getting this error:
_argument 'ids': 'list' object cannot be interpreted as an integer_

Google Colab link where I have implemented the model:
https://colab.research.google.com/drive/1gcHaM4ehqccX2zGIe8RbCeN6Q-hZh0bb?usp=sharing

Please help in this regard.

**Comment by kevinscaria** [comment]
Hey, If you want to extract the aspects and sentiment pairs for a given sentence, you have to use the joint task model weights. I have modified your colab notebook, I am not sure if you can see the changes, but I got the required answers in the notebook you shared. 

Closing this issue now. You can open it again if the issue still persists. I will also upload a sample notebook for all three subtasks of ABSA.

Cheers!

**Comment by RajNS02** [comment]
Thank you so much for your help!

Unfortunately, I am not able to see the changes. If possible, could you please share the notebook link where you made the changes?
Thanks in advance.

**Comment by kevinscaria** [comment]
Here you go:

https://github.com/kevinscaria/InstructABSA/blob/main/inference_notebook.ipynb

**Comment by RajNS02** [comment]
Thank you so much🙂
